rpc/triple/README.md

```diff
@@ -1,12 +1,90 @@
 # Triple Samples
 
-Triple-go is a network protocol library released in version 3.0 based on the existing Dubbo service management capabilities. You may refer to [dubbogo/triple](https://github.com/dubbogo/triple) for more details.
+Triple is  dubbo3.0 network protocol. You can refer to [dubbogo/triple](https://github.com/dubbogo/triple) repo for more information. Triple is extended from old Dubbogo framework, it supports pb serilization, and can interact with dubbo-java 3.0 and  gRPC. It also support unary RPC and streaming RPC invocation. Triple is main protocol that  Dubbo echology recommends.
 
-## Contents
+## Samples content
 
-- [codec-extension](./codec-extension): TODO: ~~code-extension~~
-- [hessian2](./hessian2): A Triple sample using hessian2 serialization.
-- [msgpack](./msgpack): TODO: ~~msgpack introduction~~
-- [pb](./pb): Triple samples using ProtoBuf(PB) serialization.
-    - [dubbogo-grpc](./pb/dubbogo-grpc): Samples for communication between Dubbo-go using triple protocol and original gRPC(triple-go <-> grpc).
-    - [dubbogo-java](./pb/dubbogo-java): Samples for communication between Dubbo-go and Dubbo-java using Triple protocol(triple-go <-> triple-java).
+- [codec-extension](./codec-extension): user defined serialization sample
+- [hessian2](./hessian2): Hessian2 serialization sample
+- [msgpack](./msgpack): Msgpack serialization sample
+- [pb](./pb):  ProtoBuf(PB) serialization sample
+  - [dubbogo-grpc](./pb/dubbogo-grpc): Triple and gRPC interactive samples
+  - [dubbogo-java](./pb/dubbogo-java): Triple-java and Triple-go interactive samples
+
+## How to write configuration file
+
+- Server
+
+```yaml
+dubbo:
+	protocols: # dubbogo framework network protocol configuration
+		myProtocol: # user defined protocol Key
+			name: tri # protocol name, suppporting: tri/dubbo/grpc/jsonrpc
+			port: 20000 # port to export service
+			
+	provider: 
+		services: 
+			MyProvider: # service provider type name 
+			 protocol: myProtocol # user defined protocol key，as above myProtocol defined
+			 interface: org.apache.dubbogo.MyProvider # user defined interface name
+		#  serialization: hessian2 # serialization that can be choosed：pb/hessian2/custome
+		#	 pb serialization by default

```

Comment:
Tab(\t) shouldn't be used in yaml, please consider replacing tab with 2 spaces.

Reply:
Thanks for your suggestions, Done! @xueweiniu